tool/dart_dev/config.dart

```diff
@@ -55,15 +55,15 @@ Future<int> startTestServer(DevToolExecutionContext _) async {
       mode: ProcessStartMode.inheritStdio);
   return firstOf([
     // Exit early if it fails to start,
-    _testServer.exitCode,
+    _testServer!.exitCode,
     // otherwise just wait a little bit to ensure it starts up completely.
     Future.delayed(Duration(seconds: 2)).then((_) => 0),
   ]);
 }
 
 Future<int> stopTestServer(DevToolExecutionContext _) async {
   _testServer?.kill();
-  await _testServer.exitCode;
+  await _testServer?.exitCode;
   return 0;
 }
 
```

Comment:
Down below, there's an invalid return type hint on `.catchError(c.completeError)`. Not a big deal, but it made me realize that we no longer need that `firstOf` util because we now have `Future.any()` from the SDK and it does exactly the same thing :)